webapp/README.md

```diff
@@ -0,0 +1,51 @@
+# Docker monitoring - web dashboard.
+This readme describes how to configure and run web dashboard of docker monitoring app.
+
+1. Install all requirements from root requirements.txt
+    ```
+    pip install -r requirements.txt
+    ```
+2. Complete all steps from root readme and configure postgresql database.
+3. In `app.py` tune following line:
+    * For development mode
+    ```
+    app.config.from_object('config.DevConfig')
+    ```
+    * For production mode
+    ```
+    app.config.from_object('config.ProdConfig')
+    ```
+    You can find parameters for both configs in `config.py`
+4. Web app imports some settings from environment variables. So, before running app
+you should initialize this variables.
+    * Secret key is used for secure communication between server and client via flask.
+        To initialize envirnoment variable with it, run:
+        ```
+        export SECRET_KEY="your secret key"
+        ```
+        You can configure it by yourself or generate it with following command:
+        ```
+        export SECRET_KEY="$(python -c "import os; print(os.urandom(24).hex())")"
+        ```
+    * Since the postgresql server may be located on another machine or on another
+        network, connection uri should be configured separately with such command:
+        ```
+        export DATABASE_URI="postgresql://user[:password]@[host][:port][/dbname]"
```

Comment:
What if we have no password for postgres?

Reply:
For this case password field is enclosed in square brackets, that means that this filed is optional. Format of connection URI is described in link below this line.